spacy_experimental/coref/coref_model.py

```diff
@@ -0,0 +1,381 @@
+from typing import List, Tuple, Callable, cast
+
+from thinc.api import Model, chain, get_width
+from thinc.api import PyTorchWrapper, ArgsKwargs
+from thinc.types import Floats2d, Ints2d
+from thinc.util import torch, xp2torch, torch2xp
+
+from spacy.tokens import Doc
+
+
+EPSILON = 1e-7
+
+
+def build_coref_model(
+    tok2vec: Model[List[Doc], List[Floats2d]],
+    distance_embedding_size: int = 20,
+    hidden_size: int = 1024,
+    depth: int = 1,
+    dropout: float = 0.3,
+    # pairs to keep per mention after rough scoring
+    antecedent_limit: int = 50,
+    antecedent_batch_size: int = 512,
```

Comment:
There's already an example config, so I think it should just work without these values, but I thought it was normal to include them - for example they're in the biaffine parser [here](https://github.com/explosion/spacy-experimental/blob/7ca435d01aa87b2e61760a266fe8cdb748043ee1/spacy_experimental/biaffine_parser/bilinear.py#L19). I think it's nice to have the function work on its own without requiring the config machinery, but then there's the issue of keeping the values in sync... I'll go ahead and remove these for now to at least confirm it doesn't break anything.

Reply:
Tests passed with these removed.

Reply:
Just as a note - when comparing across the code base what's "normal" I think it makes more sense to compare to stable components in the core spaCy library rather than to other new/experimental components. And correct me if I'm wrong - but since v3 I don't think we have these kind of default numbers anymore.